zkevm-circuits/src/rlp_circuit_fsm.rs

```diff
@@ -249,6 +299,17 @@ pub struct RlpCircuitConfig<F> {
     /// Boolean to reduce the circuit's degree
     is_same_rlp_instance: Column<Advice>,
 
+    /// Boolean to reduce the circuit's degree
+    is_new_access_list_address: Column<Advice>,
+    /// Boolean to reduce the circuit's degree
+    is_new_access_list_storage_key: Column<Advice>,
+    /// Boolean to reduce the circuit's degree
+    is_access_list_address_clear: Column<Advice>,
+    /// Boolean to reduce the circuit's degree
+    is_access_list_storage_key_clear: Column<Advice>,
```

Comment:
Based on the code, the `is_access_list_address_clear` is actually used to mark the end of an entire access list. So I suggest we call it `is_access_list_end` or `is_access_list_clear` instead of adding the `*_address_* as well.

Reply:
Columns renamed to `is_access_list_end` and `is_storage_key_list_end` for better semantics. Boolean constraints added.